plugins/components/conversionlayer.go

```diff
@@ -378,11 +378,21 @@ func getActionFunc(cmd Command) cli.ActionFunc {
 }
 
 func ConvertContext(baseContext *cli.Context, flagsToConvert ...Flag) (*Context, error) {
+	print(baseContext)
```

Comment:
Why is this necessary? In any case, I suggest to avoid using `print` in production.